geant-incubator/Issues/edugain-proxy/description.xml

```diff
@@ -0,0 +1,113 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo GEANT Incubator eduGAIN proxy</en>
+        <nl>Demo GEANT Incubator eduGAIN proxy</nl>
+	</Name>
+	<ShortName>
+        edugain-proxy
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>geant-incubator</IssuerID>
+	<CredentialID>edugain-proxy</CredentialID>
+	<Description>
+		<en>This issuer is part of the Distributed Identity for Research (DI4R) IRMA demo, part of the the GÉANT Trust and Identity Incubator.\nThis issuer is a proxy between Identity Providers in eduGAIN and the DI4R IRMA ecosystem. It allows users to load credetials from their home institution into an IRMA wallet for login to services lateron.</en>
```

Comment:
Missing space in `lateron` at the end of this line.

Additionally, this reads more like the description of the issuer of this credential, instead of this credential. Currently this description is not shown in the IRMA app GUI, but it might be in the future, and it is visible in the [attribute index](https://privacybydesign.foundation/attribute-index/en/). Same goes for the identical `Description` of the other credential.

Reply:
Ok I have updated this in a new pull request. I would be good (perhaps I missed this) to have a description of where which element of the schema are being used. Especially which element have a technical role and which will be exposed to end users at some point